src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -290,7 +265,7 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
                     try
                     {
                         fileTraceWriter.Info(text);
-                        traceWriter.Info(text);
+                        invocationContext.TraceWriter.Info(text);
```

Comment:
For some reason, I thought we had made the change to include it as a system parameter for .NET as well.